### PR TITLE
Minor typo fix

### PR DESCRIPTION
**Pull request by AlexXiong97**
`return abi.encode(nextState)` should belong to `applyAction` instead of `resolve`.

a minor typo which @snario confirmed weeks ago.

### CODE REVIEW

File: 02-state-machines.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Up to four functions can be implemented. The signatures are as follows:
 
 In designing the framework we must try to achieve two sometimes contradictory goals. On the one hand, we wish to allow app developers to view application state as a structured data type, the same way the developer of a non-channelized dapp would interact with contract storage. On the other hand, the framework would like to treat application state as a blob of unstructured data. Current limitations around the Solidity type system sometimes put these in conflict; for instance, we enforce the limitation that the `AppState` struct must not be dynamically sized.
 
-Another consequence is that the return type of `resolve` is actually `bytes`. We expect that application developers simply end their `resolve` function with something like
+Another consequence is that the return type of `resolve` is actually `bytes`. We expect that application developers simply end their `applyAction` function with something like
```
Comment (snario) [inline 02-state-machines.md]:
```suggestion
Another consequence is that the return type of `applyAction` is actually `bytes`. We expect that application developers simply end their `applyAction` function with something like
```

Comment (AlexXiong97) [reply 02-state-machines.md]: Good catch, I'll update this.

**Comment by AlexXiong97** [comment]
@snario the PNG changes happen because once I update the file and run `yarn build`, they will rebuild(at least touched) every files, although the actual content remained the same.

File: 02-state-machines.md (hunk 1 above)
Comment (snario) [reply 02-state-machines.md]: You can actually just click the button to auto-commit it

**Pull request merged**